sound/soc/sof/amd/Kconfig

```diff
@@ -17,6 +17,9 @@ if SND_SOC_SOF_AMD_TOPLEVEL
 config SND_SOC_SOF_AMD_COMMON
 	tristate
 	select SND_SOC_SOF
+	select SND_SOC_SOF_PCI_DEV
```

Comment:
I think you have a missing dependency on SND_SOC_SOF_PCI, which itself depends on PCI. if you look at the Intel stuff we have a 'if SND_SOC_SOF_PCI' which can also be added as 'depends on SND_SOC_SOF_PCI' for Renoir